addons/html_editor/static/src/main/media/media_dialog/file_documents_selector.js

```diff
@@ -1,41 +0,0 @@
-import { DocumentSelector } from "@html_editor/main/media/media_dialog/document_selector";
```

Comment:
Stable retrocompatibility: put this file back with a comment on top saying it is not used anymore but was kept due to the stable policy.